core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +56,13 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
```

Comment:
So, I just use the value of the configuration "alluxio.user.block.worker.client.threads" as the default factor, and also provide this value can be configurable. And use property name of "alluxio.user.file.write.capacity.reserved.ratio"?